src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerRelationship = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${partnerRelationship}`
+        ) : (
+          'I am not married'
+        )}
+      </p>
```

Comment:
```suggestion
      <p className="Person__partner">
        {isMarried 
          ? `${partnerName} is my ${partnerRelationship}`
          :  'I am not married'
        }
      </p>
```
![image](https://github.com/mate-academy/react_person/assets/96409563/01e3b63c-f35f-42d1-8b37-0680b42bde4f)